client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -161,6 +161,24 @@ export function ScoreTable(props: Props) {
     return null;
   }
 
+  function trimFiltersProps(filters: ScoreTableFiltersModified): ScoreTableFiltersModified {
```

Comment:
could you move that helper function outside of the component?

Reply:
ok I did it.